docs/reference/mapping/types/ip.asciidoc

```diff
@@ -156,3 +156,46 @@ GET my-index-000001/_search
   }
 }
 --------------------------------------------------
+
+ifeval::["{release-state}"=="unreleased"]
+[[ip-synthetic-source]]
+==== Synthetic source
+`ip` fields support <<synthetic-source,synthetic `_source`>> in their
+default configuration. But they don't support synthetic `_source` if you add
+<<ignore-malformed,`ignore_malformed`>>, <<copy-to,`copy_to`>>, or if you
+disable <<doc-values,`doc_values`>>.
+
+Synthetic source will always sort `ip` fields and remove duplicates so:
+[source,console,id=synthetic-source-ip-example]
+----
+PUT idx
+{
+  "mappings": {
+    "_source": { "synthetic": true },
+    "properties": {
+      "ip": { "type": "ip" }
+    }
+  }
+}
+PUT idx/_doc/1
+{
+  "ip": ["192.168.0.1", "192.168.0.1", "10.10.12.123",
+         "2001:db8::1:0:0:1", "::afff:4567:890a"]
+}
+----
+// TEST[s/$/\nGET idx\/_doc\/1?filter_path=_source\n/]
+
+Will become:
+
+[source,console-result]
+----
+{
+  "ip": ["::afff:4567:890a", "10.10.12.123", "192.168.0.1", "2001:db8::1:0:0:1"]
+}
+----
+// TEST[s/^/{"_source":/ s/\n$/}/]
+
+NOTE: IPv4 addresses are sorted as though they were IPv6 addresses prefixed by
+      `::ffff:0:0:0/96` ala https://datatracker.ietf.org/doc/html/rfc6144[rfc6144].
```

Comment:
```suggestion
NOTE: IPv4 addresses are sorted as though they were IPv6 addresses prefixed by
      `::ffff:0:0:0/96`, following
      https://datatracker.ietf.org/doc/html/rfc6144[rfc6144] guidelines.
```

Reply:
I'm not sure I'd call this a "guideline". Maybe I should do, but I think of it more like a "regime" or "as specified by rfc6144.

Reply:
I think `as specified by` is perfect!